.jenkins/jenkins_pr_win.bat

```diff
@@ -0,0 +1,51 @@
+REM CUDNN PATH
```

Comment:
Duplicate from what? This is the comment explaining the CUDNNPATH variable, which is used further down.